app/models/photo.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Photo < ApplicationRecord
```

Comment:
Maybe it would be a good idea to extract all the telegram logic into a separate class at some point? Right now, it’s scattered across controllers and models.

```ruby
class TelegramMessage
  def initialize(message)
    @message = message
  end

  def create_reply!
    # create a Reply recors
  end

  def download_photos!
    # download photos from Telegram API and attach them to reply
  end

  private

  def user
    # find user by chat id
  end
end
```

Reply:
This might even provide a common interface for incoming messages of all channels:

```ruby
class EmailMessage
  def initialize(email)
    @email = email
  end

  def create_reply!
    # ...
  end

  def create_photos!
    # ...
  end

  private

  def user
    # find user by email address
  end
end
```